lib/SearchAndSort/SearchAndSort.js

```diff
@@ -383,13 +385,14 @@ class SearchAndSort extends React.Component {
 
   transitionToParams = values => {
     const {
+      location,
+      history,
       nsParams,
-      parentMutator: { query },
     } = this.props;
 
     const nsValues = mapNsKeys(values, nsParams);
-
-    query.update(nsValues);
+    const url = buildUrl(location, nsValues);
+    history.push(url);
```

Comment:
Could you please have an empty line after the definition of variables. I have such setup in my project and followed this pattern while cleaning up the S&S component. I also ensure the blank line before the `return` construction. This is only for readability purposes.

Reply:
@VictorSoroka1 thank you for reviewing it. Yes of course I will change it. Also are you using Visual Studio Code? If so I would love to see where you can configure it.

Reply:
I use the following `eslint` rule:

```
    "padding-line-between-statements": [
      "error",
      { "blankLine": "always", "prev": ["const", "let"], "next": "*"},
      { "blankLine": "any", "prev": ["const", "let"], "next": ["const", "let"]},
      { "blankLine": "always", "prev": "*", "next": ["return"]}
    ],
```

I understand that this is not a part of that project so adding this may end up in a bunch of lint errors.

Reply:
Thanks. Do you maintain a global .eslintrc?

Reply:
I am extending this project but add several rules on top of it in my project.

Reply:
Thank you. I was able to set them up. Now I'm getting over 100 errors :) but I will fix this one.

Reply:
Thanks 😄.